### PR TITLE
Fix broken links in references

### PR DESCRIPTION
**Pull request by gpasztor87**

### CODE REVIEW

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf [hunk 1]
```diff
@@ -23,7 +23,6 @@ SecRule TX:PARANOIA_LEVEL "@lt 1" "phase:2,id:941012,nolog,pass,skipAfter:END-RE
 #
 # -=[ Libinjection - XSS Detection ]=-
 #
-# Ref: https://libinjection.client9.com/
```
Comment (csanders-git) [inline rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:26]: This should probably link to their github
https://github.com/client9/libinjection

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf [hunk 2]
```diff
@@ -702,7 +701,6 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 	setvar:tx.%{rule.id}-OWASP_CRS/WEB_ATTACK/XSS-%{matched_var_name}=%{tx.0}"
 
 #
-# http://openmya.hacker.jp/hasegawa/security/utf7cs.html
```
Comment (csanders-git) [inline rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:705]: That's a shame this is down (http://web.archive.org/web/20170126151511/http://openmya.hacker.jp/hasegawa/security/utf7cs.html) maybe we should mirror it?

**Comment by csanders-git** [comment]
Overall this looks really good. Probably the only thing i'd actually update would be the libinjection link. The rest look great. nice job finding replacements where needed. After that its good for a merge i think thanks @gpasztor87. Let me know if you're willing to do that one change

**Comment by gpasztor87** [comment]
I fixed that one link. :)

**Comment by csanders-git** [comment]
Looks good to me :)

**Comment by csanders-git** [comment]
Wow was so close to merging, just realized you did this against v3.0/master. Any chance you can rebase against v3.0/dev? Let me know otherwise i'll move it for you, thanks for the contribution again.

**Comment by lifeforms** [comment]
*v3.1/dev 🐙

**Pull request closed without merging**